app/controllers/org_admin/templates_controller.rb

```diff
@@ -292,11 +292,13 @@ def publish
     def unpublish
       template = Template.find(params[:id])
       authorize template
-      versions = Template.where(family_id: template.family_id)
-      versions.each do |version|
-        unless version.update_attributes!(published: false)
-          flash[:alert] = _("Unable to unpublish your #{template_type(template)}.")
+      Template.transaction do
+        # expected: template is latest
+        if template.published? && template.plans.any?
```

Comment:
Change this to a one-liner `template.generate_version! if template.published? && template.plans.any?` to make Rubocop happy